functests/3_performance_status/status.go

```diff
@@ -0,0 +1,75 @@
+package __performance_status
+
+import (
+	"context"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	testutils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+	testclient "github.com/openshift-kni/performance-addon-operators/functests/utils/client"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/mcps"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/nodes"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/profiles"
+	v1 "github.com/openshift/custom-resource-status/conditions/v1"
+	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("Status testing of performance profile", func() {
+	var workerCNFNodes []corev1.Node
+	var err error
+
+	BeforeEach(func() {
+		workerCNFNodes, err = nodes.GetByRole(testutils.RoleWorkerCNF)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(workerCNFNodes).ToNot(BeEmpty())
+	})
+
+	Context("[rfe_id:28881][performance] Performance Addons detailed status", func() {
+		var currentConfig string
+		nodeAnnotationConfig := "machineconfiguration.openshift.io/currentConfig"
+		nodeLabel := map[string]string{fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerCNF): ""}
+
+		annotations := workerCNFNodes[0].GetAnnotations()
```

Comment:
You need to move this inside `It()`: `BeforeEach()` is executed just before all the `It()`s, so it didn't run at this place yet. That explains the `index out of range`. I didn't see it in earlier reviews.